ruby/spec/routes_spec.rb

```diff
@@ -2,41 +2,42 @@
 require 'rack/test'
 require 'rspec/autorun'
 require 'json'
-require_relative '../app/routes/health'
+require 'base64'
+require_relative '../app/middleware/auth'
+require_relative './helpers/token'
 require_relative '../app/routes/home'
-require_relative './stubs/stub_middleware'
 require_relative '../app/routes/articles'
 
-
 describe HomeRoutes do
   include Rack::Test::Methods
 
   let(:app) { HomeRoutes.new }
 
   context 'testing the homepage endpoint GET /' do
-    let(:response) { get '/' }
+    let(:response) { get '/', nil, {'HTTP_AUTHORIZATION' => "Basic #{create_token}" } }
 
     it 'checks response status' do
       expect(response.status).to eq 200
     end
   end
 end
 
-
 describe ArticleRoutes do
   include Rack::Test::Methods
 
   let(:app) { ArticleRoutes.new }
+  let(:token) {create_token}
```

Comment:
```suggestion
  let(:basic_auth_header) { auth_header_content }
```

Reply:
I'd rather do `let(:auth_header) {auth_header_content}`. No need to reveal what type of auth header this is in case we update it at some point in the future